Jenkinsfile

```diff
@@ -0,0 +1,74 @@
+#!groovy
+
+@Library('jenkins-utils-lib') _
+
+def devBranch = "develop"
+def prodBranch = "master"
+
+def map_branches = [
+	'^master$': 'focal-agent',
+	'^develop$': 'focal-agent-dev',
+	'.*': 'focal-agent-dev',
+	]
+
+pipeline
+{
+	agent
+	{
+		label 'dr-linbuild'
+	}
+	options
+	{
+		buildDiscarder(logRotator(numToKeepStr: '10'))
```

Comment:
I think we need add here a function to use number of builds to keep depends on branch type, like we did it for agent:
https://github.com/Axcient/agent/blob/41f06419e54e6405f0ac5bcb0d5dc22a84d89a01/Jenkinsfile#L13

Reply:
This change is unrelated to RA-2585. Let's do it in scope of another PR.